TYPO3.TYPO3CR/Classes/TYPO3/TYPO3CR/Domain/Model/NodeInterface.php

```diff
@@ -293,17 +293,6 @@ public function setAccessRoles(array $accessRoles);
     public function getAccessRoles();
 
     /**
-     * Sets the absolute path of this node
-     *
-     * This method is only for internal use by the content repository. Changing
-     * the path of a node manually may lead to unexpected behavior and bad breath.
-     *
-     * @param string $path
-     * @return void
-     */
-    public function setPath($path);
```

Comment:
The method was never part of the public API and even states it should not be used. It is purely internal and therefore protected in Node now. That allows removal from the interface which I do not considered breaking.